tests/test_geoprocessing.py

```diff
@@ -2587,7 +2587,8 @@ def test_convolve_2d_single_thread(self):
         # the sides and realizing diagonals got subtracted twice
         expected_result = test_value * (
             n_pixels ** 2 * 9 - n_pixels * 4 * 3 + 4)
-        self.assertAlmostEqual(numpy.sum(target_array), expected_result)
+        self.assertAlmostEqual(numpy.sum(target_array), expected_result,
+                              places=3)

```

Comment:
I can't quite tell, is the indent off by one space?

```suggestion
                               places=3)
```

Reply:
Huh, no idea how that happened!  Patched in d916244.